test_app/tests/lib/utils/test_models.py

```
@@ -313,6 +313,74 @@ def test_diff_sanitizes_encrypted_fields_removed(disable_activity_stream):
     assert 'message' not in delta.removed_fields
 
 
+@pytest.mark.django_db
+def test_diff_sanitizes_encrypted_value_prefix_changed(disable_activity_stream):
```

Ruff (0.15.2)

[warning] Unused function argument: `disable_activity_stream`

(ARG001)